airbyte-integrations/connectors/destination-s3-v2/build.gradle

```diff
@@ -5,12 +5,13 @@ plugins {
 
 airbyteBulkConnector {
     core = 'load'
-    toolkits = ['load-aws', 'load-s3']
+    toolkits = ['load-aws', 'load-object-storage', 'load-s3']
```

Comment:
low confidence: should `load-s3` use `api project("load-object-storage")` instead of `implementation`, so that you automatically get load-object-storage alongside it?

(I honestly have no idea what's considered good practice here, it just seems a little weird that we have to manually declare the s3 module _and_ its dependencies. This is also incredibly bikesheddy, since we're basically never going to edit this line :shrug: )

Reply:
Hm. The declaration is because it uses some object storage stuff directly, however that stuff is like "declare my config a provider of a type that the s3 stuff returns" so it makes sense to pull it in with load-s3.

Reply:
Yeah, it's better that way. I did the same thing for `load-aws`, so now this is just `load-s3`, which brings in the key config and object storage generic stuff.